src/pebble.py

```diff
@@ -22,6 +22,7 @@
 logger = logging.getLogger(__name__)
 
 STATS_EXPORTER_SERVICE_NAME = "stats-exporter"
+MAS_CONFIGURATION_PATH = "/mas/config.yaml"
```

Comment:
Consider adding to the storage /data (reminder how to deal with assets?)

Reply:
I think we still need further discussion on this, so let's move this topic internally and I'll create a PR to follow-up if needed.